src/stylesheets/components/_alerts.scss

```diff
@@ -3,40 +3,40 @@ $usa-custom-alerts: () !default;
 $usa-custom-alerts-bar: () !default;
 
 $usa-alerts: (
-  success: $color-green-lightest,
-  warning: $color-gold-lightest,
-  error: $color-secondary-lightest,
-  info: $color-primary-alt-lightest
+  success: color('success-lighter'),
+  warning: color('warning-lighter'),
+  error: color('error-lighter'),
+  info: color('accent-cool-lighter')
 );
 
 $usa-alerts-bar: (
-  success: $color-green,
-  warning: $color-gold,
-  error: $color-secondary,
-  info: $color-primary-alt
+  success: color('success'),
+  warning: color('warning')
+  error: color('error'),
+  info: color('accent-cool')
 );
 
 $alerts: map-merge($usa-alerts, $usa-custom-alerts);
 $alerts-bar: map-merge($usa-alerts-bar, $usa-custom-alerts-bar);
 
-$h-padding: $spacing-medium;
-$bar-size: $spacing-small;
+$h-padding: units($theme-spacing-md);
+$bar-size: units($theme-spacing-xs);
 $left-padding: $h-padding + $bar-size;
-$icon-size: 3rem;
+$icon-size: units($theme-spacing-lg);
 
 .usa-alert {
   background-color: $color-gray-lightest;
   background-position: $h-padding $h-padding;
   background-repeat: no-repeat;
-  background-size: 5.2rem; // TODO abrituary number based on svg size maybe?
-  padding-bottom: $h-padding * 0.7;
+  background-size: units(7);
+  padding-bottom: units($theme-spacing-sm);
   padding-left: $left-padding;
   padding-right: $h-padding;
   padding-top: $h-padding;
   position: relative;
 
   &::before {
-    background-color: darken($color-gray-lightest, 40%);
+    background-color: color('gray-40');
```

Comment:
I ended up going with the specific grade of gray. I don't actually know when this is relevant, since all the alerts are overriding the background color with their own. I guess it might come into play if someone uses a `usa-alert` with no modifier?